uefi-raw/src/protocol/scsi.rs

```diff
@@ -80,17 +80,75 @@ newtype_enum! {
 #[derive(Debug, Copy, Clone)]
 #[repr(C)]
 pub struct ScsiIoScsiRequestPacket {
+    /// The timeout, in 100 ns units, for the execution of this SCSI Request Packet.
+    ///
+    /// A `timeout` value of 0 indicates that the function will wait indefinitely for
+    /// the execution to complete. If the execution time exceeds the specified `timeout`
+    /// (greater than 0), the function will return `EFI_TIMEOUT`.
```

Comment:
nit: could be `[``TIMEOUT``]` linking to https://docs.rs/uefi-raw/latest/uefi_raw/struct.Status.html#associatedconstant.TIMEOUT